tests/ethercat/test_ethercat_servo.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from ingenialink.exceptions import ILStateError
+
+
+@pytest.mark.ethercat
+def test_emcy_callback(mocker, connect_to_slave):
+    servo, _ = connect_to_slave
+    # Mocker does not detect the calls to ingenialink.ethercat.network.EthercatNetwork._emcy_callback
+    # so we use this function instead.
+    mocked = mocker.patch("ingenialink.ethercat.servo.EthercatServo.get_emergency_description")
+    prev_val = servo.read("DRV_PROT_USER_OVER_VOLT", subnode=1)
+    servo.write("DRV_PROT_USER_OVER_VOLT", data=10.0, subnode=1)
+    with pytest.raises(ILStateError):
+        servo.enable()
+    servo.fault_reset()
+    assert mocked.call_count == 2
```

Comment:
Why is it called two times? Does the mocker callback record the arguments (description) it was called with?

Reply:
The drive sends two emergency messages, one when the fault occurs and one when the fault is cleared (fault reset).